imu_processor:
  ros__parameters:
    # 角速度协方差 (必须全部使用浮点表示)
    angular_velocity_covariance: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    
    # 线加速度协方差
    linear_acceleration_covariance: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    
    # 姿态协方差
    orientation_covariance: [99999.9, 0.0, 0.0, 0.0, 99999.9, 0.0, 0.0, 0.0, 99999.9]
    
    # 字符串参数
    frame_id: "imu_link"

    # 零飘检测
    calibrate_flag: false